scripts/utils/import-metrics/import-metrics.sh

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+
+# Script using mimirtool for prometheus tsdb collection
+# 
+# Usage: ./import-metrics.sh /path/to/kubeconfig.yaml selector from to offset
+#
+#   arg: path to kubeconfig (required via cli or as environment variable)
+#   arg: prometheus query selector (optional)
+#   arg: target date for query to start from (optional)
+#   arg: target date for query to end (optional)
+#   arg: offset(in seconds) (optional)
+#
+# See README for more usage information
+
+# offset_seconds - query time range loop increment, only modify if default prometheus installation memory has been increased, default is set for one hour
+offset_seconds=3600 # one hour
+
+# selector - a valid prometheus query in single quotes, default selector set for ALL METRICS
+selector='{__name__!=""}'
+
+# from - date for query to begin
+# to - date for query to end
+# default time range set for ONE HOUR from current utc time
+from="$(date -u -v-1H +"%Y-%m-%dT%H:%M:%SZ")" 
```

Comment:
I think we should not rely on flags/behaviors that only work on 1 OS (e.g. this `-v` flag only exists on Mac OS' date command). You could probably work around this by checking for what `uname` returns, though.

Reply:
Also, this flag is outputting the date from 1 hour ago, not 1 hour *from* current time.

Reply:
As per our discussion, updated date commands based on os using uname 

Sorry for the confusion on the context of "from", I would prefer start/end for clarity but from/to comes from the usage flags on `mimirtool remote-read export` so I carried it over to be transparent. Default behavior if only one date is provided is to utilize the date as the "from" date and set the "to" date to current time. So, in plain english, we're _not_ aiming to capture "one hour from now", we want "from one hour ago" (to now), if that makes sense.